src/main/kotlin/org/malachite/estella/task/api/TaskController.kt

```diff
@@ -62,6 +65,19 @@ class TaskController(
     private fun isPasswordProvided(organizationId: String?, taskStageUuid: String?, devMail: String?, password: String?): Boolean =
             listOfNotNull(organizationId, taskStageUuid, devMail).isNotEmpty() && password != null
 
+    @CrossOrigin
+    @Transactional
+    @GetMapping("/forJobSeeker")
+    fun getTasksForJobSeeker(
+            @RequestParam("taskStage") taskStageUuid: String?
+    ): ResponseEntity<Any> {
+        if (taskStageUuid == null)
+            return ResponseEntity.badRequest().body(Message("taskStageUuid is required"))
```

Comment:
To skoro jest required, to daj bez pytajnika XD Wtedy samo rzuci bad request.

Reply:
tak, właśnie ogarnąłem jak dodawałem drugi parametr XDD